src/vm/object_model.rs

```diff
@@ -379,6 +379,24 @@ pub trait ObjectModel<VM: VMBinding> {
     /// mature space for generational plans.
     const VM_WORST_CASE_COPY_EXPANSION: f64 = 1.5;
 
+    /// If this is true, the binding guarantees that an object reference's raw address equals to the return value of the ref_to_address method,
+    /// and equals to the return value of the ref_to_object_start method. This is a very strong guarantee, but it is also helpful for MMTk to
```

Comment:
It is "A equals B" or "A is equal to B".

```suggestion
    /// If this is true, the binding guarantees that an object reference's raw address is always equal to the return value of the `ref_to_address` method
    /// and the return value of the `ref_to_object_start` method. This is a very strong guarantee, but it is also helpful for MMTk to
```

Reply:
Done.